CONTRIBUTING.md

```diff
@@ -207,19 +209,27 @@ Look inside a table
     ----+-----------+-----------
     (0 rows)
 
+### ER Diagram
+
+There is an [ER Diagram](files/ERDiagram.png), but BEWARE it might be outdated
```

Comment:
What about displaying the image here? Or, is that too distracting?

Reply:
I thought it's quite big, but sure, why not.